sdk/appcenter-identity/src/test/java/com/microsoft/appcenter/identity/IdentityTest.java

```diff
@@ -177,8 +177,8 @@ private void testInvalidConfig(JSONObject jsonConfig) throws Exception {
         verifyStatic();
         FileManager.write(any(File.class), anyString());
 
-        /* Check we didn't try login after configuration attempt. */
-        assertTrue(identity.isLoginDelayed());
+        /* Check we didn't try signIn after configuration attempt. */
```

Comment:
In the docs we used **sign-in** for text instructions. In the comments and Javadoc we should also use sign-in with a dash.

Reply:
sign-in is a noun.  Here it's used as a verb, which is unhyphenated (`check we didn't try to sign in after`...)

Reply:
but we should still split the word

```suggestion
        /* Check we didn't try to sign in after configuration attempt. */
```